### ISSUE TITLE
训练数据集

### ISSUE THREAD

**Issue by YCG09** [xiaomaxiao/keras_ocr#2]
你好，请问训练数据是用SynthText生成的吗？可否提供下训练集呢，谢谢

**Comment by xiaomaxiao** [comment]
这个 https://github.com/senlinuc/caffe_ocr

**Comment by YCG09** [comment]
谢谢，还想问下weights-51.hdf5这个模型什么训练出来的呢？

model.load_weights(r'E:\deeplearn\OCR\Sample\model\weights-51.hdf5')

**Comment by xiaomaxiao** [comment]
我是tf的后端 ，具体模型不方便发出来，你用上面链接的数据 也可以训练。效果也挺好的。

**Comment by YCG09** [comment]
好的，我也是tf的后端，主要是因为调整模型结构后原本的pretrain模型就不能用了，所以想问问

**Comment by YCG09** [comment]
你好，我用上面链接的数据训练时，loss只能下降到70左右，acc一直是0，问题找了半天也没解决，请问你觉得可能是什么原因啊？

**Comment by xiaomaxiao** [comment]
查下样本标签是不是对的。
欢迎入群交流274634783

**Comment by YCG09** [comment]
之前是由于觉得360万数据量太大，怕训练不动，所以随机抽了5万训练，现在用95%数据训练loss可以下降到0.2，但是acc只有2%-3%，不知道什么原因（batchsize=256，lr=0.001）

13522/13522 [==============================] - 35462s - loss: 8.9631 - acc: 0.0878 - val_loss: 0.2725 - val_acc: 0.0463
Epoch 2/30
13522/13522 [==============================] - 34625s - loss: 0.2169 - acc: 0.0324 - val_loss: 0.2540 - val_acc: 0.0347
Epoch 3/30
13522/13522 [==============================] - 34620s - loss: 0.2006 - acc: 0.0268 - val_loss: 0.1971 - val_acc: 0.0276
Epoch 4/30
13522/13522 [==============================] - 34609s - loss: 0.1833 - acc: 0.0236 - val_loss: 0.2269 - val_acc: 0.0257
Epoch 5/30
13522/13522 [==============================] - 34625s - loss: 0.1841 - acc: 0.0225 - val_loss: 0.1918 - val_acc: 0.0232
Epoch 6/30
13522/13522 [==============================] - 34626s - loss: 0.1957 - acc: 0.0230 - val_loss: 0.2154 - val_acc: 0.0229
Epoch 7/30
10127/13522 [=====================>........] - ETA: 8570s - loss: 0.2038 - acc: 0.0236

**Comment by xiaomaxiao** [comment]
模型是没问题的， 你看数据标签是不是对的

**Comment by jewelcai** [comment]
> 你好，我用上面链接的数据训练时，loss只能下降到70左右，acc一直是0，问题找了半天也没解决，请问你觉得可能是什么原因啊？

请问后来怎么解决的呢？
